docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Reviews Rating
+---
+
+The following documentation goes through an example implementation of a review rating platform for the food service industry on Sui. By leveraging blockchain to allow reviewers to own and monetize their feedback, the on-chain platform ensures transparency and temper-proof ratings.
+
+## Personas
+
+There are four actors in the typical workflow of the reviews rating example.  
+
+```mermaid
+sequenceDiagram
+  Service ->> Dashboard: Add service to dashboard.
+  Service ->> User: Send proof of experience.
+  User ->> Service: Send review
+  Service ->> User: Send reward
+  Service ->> Moderator: Add as a service moderator
+  Moderator ->> Service: Remove abused review
+```
+
+### Service owners
+
+Service owners are entities like restaurants that list their services on the platform. Service owners allocate a specific amount of SUI as a reward pool. Assets from the pool are used to provide rewards for honest reviews. A Proof of Experience (PoE) NFT confirms the reviewer used the service, which they burn to provide a verified review. Service owners can provide their customers with unique identifiers, perhaps using QR codes, to identify individual reviewers.
+
+### Reviewers
+
+Reviewers are consumers of services that use the review system. Reviewers provide feedback in the form of comments that detail specific aspects of the service as well as a star rating to inform others. The reviews are rated, with the most effective reviews getting the highest rating. Service owners award the 10 highest rated reviews for their service. 
+
+### Review readers
+
+Review readers access reviews to make informed decisions on selecting services. Readers rate reviews by casting up or down votes. The review readers' ratings are factored into the algorithm that rates the reviews, with the authors of the highest-rated reviews getting awarded. 
+
+### Moderator
+
+The moderator monitors content of the reviews and can delete any reviews that contain inappropriate content.
```

Comment:
Couple of questions I have reading through this section:

- How are we defining an "honest" review?
- What is stopping a Service owner, masquerading as a Review reader in order to downvote unfavourable reviews (to avoid having to reward them?), or as a Moderator to remove them entirely, or as a Reviewer to create artificially inflated reviews?

Reply:
Further questions I had after reading the guide:

- How do you identify Moderators and Service Owners in the smart contract (because they can perform privileged operations).
- How do pay-outs work? E.g. is it on a regular or rolling basis, are the rewards staked by the service owner up-front or after the fact, etc?

Reply:
* I've removed "honest"
* It is true that a service owner may impersonate a reviewer, or make one self a moderator and delete all unfavorable reviews. But since all activities are on-chain I was hoping that any such unsavory actions can be uncovered later which would bring dire consequences for service owners (i.ei. banned from listing their service). 
* Another aspect I want to highlight is for these online platforms we are actually very reliant on moderator to keep things in order: delete obscene material, kick out bad actors, etc. Think about how invaluable moderators are for Mastodon or Discord. As such, I expect that moderators are ones who should be safekeepers until we can somehow integrate ChatGpt to rate reviews.
* Added a section to describe AdminCaps given to service owners and moderators
* Added a line to describe that pay-outs is all up to service owners: they can choose to pay regular or rolling basis